packages/web-components/src/components/ic-skip-link/ic-skip-link.css

```diff
@@ -0,0 +1,23 @@
+ic-link {
+  position: absolute;
+  left: -9999px;
+}
+
+ic-link:focus {
+  z-index: var(--ic-z-index-skip-link);
+  top: var(--ic-space-sm);
+  left: var(--ic-space-sm);
+  padding: var(--ic-space-sm);
+  box-shadow: var(--ic-border-focus);
+  border-radius: var(--ic-border-radius);
+}
+
+.full-width:focus {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
```

Comment:
Will this styling allow it to be used in tree view?

Reply:
Should do now. Have tested with a few elements on the page.